jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -126,6 +126,28 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    override fun send(packetInfo: PacketInfo) {
+        when (val packet = packetInfo.packet) {
+            is VideoRtpPacket -> {
+                if (bitrateController.transformRtp(packetInfo)) {
+                    // The original packet was transformed in place.
+                    transceiver.sendPacket(packetInfo)
```

Comment:
This sends the packet twice? The original has a `return`

Reply:
Yeah, I found this and fixed it in a later commit.  You could've saved me some debugging time ;)